external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSourceProvider.scala

```diff
@@ -562,6 +562,7 @@ private[kafka010] object KafkaSourceProvider extends Logging {
     "startingoffsetsbytimestampstrategy"
   private val GROUP_ID_PREFIX = "groupidprefix"
   private[kafka010] val INCLUDE_HEADERS = "includeheaders"
+  private[kafka010] val MOCK_SYSTEM_TIME = "mockSystemTime"
```

Comment:
It might be good to have a prefix that this is internal and test only, something like `__` or `testonly-` or any better prefix?
Leaving a code comment saying "this is test-only internal and won't be documented" would help as well.

Reply:
sure